flink-libraries/flink-table/src/test/scala/org/apache/flink/api/table/sink/CsvTableSinkTest.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.api.table.sink
+
+import java.io.File
+import java.nio.charset.Charset
+import java.nio.file.Files
+import java.util.Collections
+
+import org.apache.flink.api.java.ExecutionEnvironment
+import org.apache.flink.api.java.operators.DataSource
+import org.apache.flink.api.table.Row
+import org.apache.flink.api.table.sinks.CsvTableSink
+import org.junit.rules.TemporaryFolder
+import org.junit.{Rule, Test}
+import org.junit.Assert._
+
+class CsvTableSinkTest {
```

Comment:
TableSinkITCase class seems to be an integration test (expensive and high level), while my intention was to create a unit-test specifically for CsvTableSink (cheap and low-level) to make sure I don't break it with my changes. Therefore I thought it is better to keep them separate.

Do you think it's better to merge them and turn unit tests into integration tests?